compiler/rustc_trait_selection/src/traits/coherence.rs

```diff
@@ -353,49 +353,75 @@ fn negative_impl<'cx, 'tcx>(
     })
 }
 
+/// Try to prove that a negative impl exist for the given obligation and their super predicates.
```

Comment:
```suggestion
/// Try to prove that a negative impl exists for the given obligation and their super predicates.
```

"their super predicates," or "its super predicates"?

Reply:
@pierwill, thanks for the fix suggestions. I'm going to open another PR in a few, about something else, if you didn't provide a PR for these fixes I can try to add them there.